### DISCUSSION TITLE
Installing from a USB drive

### DISCUSSION THREAD

**Discussion by jlaggui-lag** [grahampugh/erase-install#98] [Q&A]
How would you change the code around so that instead of having to download the installer, we can point it to a USB with the installer in there? This would cut down on time tremendously.

**Comment by grahampugh** [comment]
You don't need to change any code for that. If you have the app in a different location, use the `--path` parameter. for example, here I have an installer app in `~/installers`, but it could be `/Volumes/MYDISK`:

```
% sudo /Library/Management/erase-install/erase-install.sh --path ~/installers 

   [erase-install] v0.20.1 script execution started: Mon Jun  7 17:02:00 CEST 2021
   [erase-install] Caffeinating this script (pid=83219)
   [erase-install] Looking for existing installer
   [find_existing_installer] Installer found at /Users/gpugh/installers/Install macOS Big Sur.app.
...
```

**Reply by jlaggui-lag** [reply]
Thanks for the help. This tool has been a huge help in our office's workflow!
